packages/big-design/src/components/Table/HeaderCell/HeaderCell.tsx

```diff
@@ -61,7 +63,7 @@ const InternalHeaderCell = <T extends TableItem>({
       width={width}
       stickyHeight={actionsSize.height}
     >
-      <StyledFlex alignItems="center" flexDirection="row" align={align}>
+      <StyledFlex alignItems="center" flexDirection="row" hide={hide} align={align}>
```

Comment:
Just a 💭, but do you think we should disable sorting functionality if a header is visually hidden? Are there any use-cases in which we might want to sort when it's visually hidden?

Reply:
I thought about it but couldn't come to a definite decision. Probably a good question for design, but I can see the possibility of wanting to sort a column that doesn't have a label. Might be good to have a group discussion around this.

Reply:
Per conversation with design + others, simplest solution for the time being is to continue to allow sorting and to render the sort arrow icon in order to make that functionality appear intentional rather than a bug. If, in the future, this solution no longer meets the needs of users or causes problems, it will be addressed then.